src/easy_log.rs

```diff
@@ -0,0 +1,28 @@
+use super::prelude;
+
+/// Set the logs verbosity based on an integer value:
+///
+/// - `0`: error
+/// - `1`: warn
+/// - `2`: info
+/// - `3`: debug
+/// - `>=4`: trace
+///
+/// This is used in the [`main!`] macro. You should typically use that instead.
+///
+/// [`main!`]: macro.main.html
+pub fn set_log_verbosity(verbosity: u64) -> prelude::Result<()> {
```

Comment:
I like that.

I'm not sure what `default_level` is. Is that an argument? How would it be used? I currently like the behavior that `verbosity=0` gives you the default of error logging. If you don't want _any_ logging than just don't call `set_log_verbosity`!

Reply:
Actually, I retract that comment. Let's either

- change this to a thing that does the mapping from int to log level (basically return `log_level` and skip the rest)
- or to a more complicated thing with a builder like `Logger::new().this_crate(2).other_crates("Warn").init()?` (note it's generic over ints and strings) and go all in, with the option of adding more cool stuff later on (e.g. detect that this is run without a tty and write to syslog/journald)

The former is a bit conservative and requires you to write the LoggerBuilder call with the two filters yourself. It also gives you more flexibility that way, of course. The latter is way more complicated, but might allow using quicli in more contexts.

FYI, I haven't seen an _easy_ logging library yet that gives us more than env_logger. Most others require setting up a bunch of different adaptors.

Reply:
I actually think `env_logger` is the right choice for this library. I didn't realize it lets you control the logging through _both_ a specified verbosity _and_ env-vars.

Hmm... I think that is providing too much complexity. What if there was a `other_verbosity: Option<u64>` which by default would do the behavior you defined in #46, but would allow you to do something like `other_verbosity: Some(u64::min(cmd.verbosity, 2))`.

I think kicking off the logging is a huge benefit here. I don't think `quicli` is the right place to solve logging ergonomics.